source/includes/container/steps-configure-minio-kes-hashicorp.rst

```diff
@@ -53,9 +53,7 @@ a. Create the KES Configuration File
             -v |kescertpath|/certs:/certs                                \
             kes:|kes-stable| tool identity of /certs/minio-kes.cert
 
-   - Replace the ``vault.endpoint`` with the hostname of the Vault server(s).
-
-   - Replace the ``VAULTAPPID`` and ``VAULTAPPSECRET`` with the appropriate :ref:`Vault AppRole credentials <minio-sse-vault-prereq-vault>`.
+   - Refer to the instructions for setting up KES for your :kes-docs:`supported KMS solution <#kes-supported-targets>` for other variables to define.
```

Comment:
Are these `other variables` specific for each KMS? Maybe something along the lines of `additional variables for ...` would work.

Reply:
Yep, expanded the wording some here to clarify.